### PR TITLE
Add date and currency formatter

### PR DESCRIPTION
**Pull request by budiariyanto**
## What does this PR do?
Add formatter for IDR currency and date time (Asia/Jakarta) 

## Why are we doing this? Any context or related work?
This function used everywhere when generating notification.

## Where should a reviewer start?
--
## Screenshots
--

## Manual testing steps?
--

## Database changes
--

## Config changes
--

## Deployment instructions
--

### CODE REVIEW

**Review by cikupin — CHANGES_REQUESTED** [review]
@budiariyanto unit testnya kenapa gak pake assertion ya mas?

File: text/formatter_test.go [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+	FormatIDR(1000000)
+
+	// Output: Rp1.000.000
+}
+
+func TestTranslateDateToIndonesian(t *testing.T) {
+	date := time.Date(2020, time.October, 6, 10, 46, 03, 0, time.UTC)
+	TranslateDateToIndonesian(date)
+
+	// Output: Selasa, 6 Oktober 2020, 17:46 WIB
```
Comment (ardityawahyu) [inline text/formatter_test.go:18]: test nya ngga pake assert sekalian mas?

**Comment by budiariyanto** [comment]
> @budiariyanto unit testnya kenapa gak pake assertion ya mas?

@cikupin O iya. Haha. Efek lelah ini mungkin wkwk. Ganti dulu

File: text/formatter_test.go (hunk 1 above)
Comment (budiariyanto) [reply text/formatter_test.go]: @ardityawahyu updated ya dit

**Review by ardityawahyu — APPROVED** [review]

**Review by cikupin — APPROVED** [review]

**Pull request merged**